python/search_completer/global_search_result_delegate.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import sgtk
+from sgtk.platform.qt import QtCore
+
+from .search_result_delegate import SearchResultDelegate
```

Comment:
Code from this class was taken from the original implementation of the `SearchResultDelegate`. The code itself hasn't changed, expect that it now underlines the search terms in the result.